pytest_fixtures/api_fixtures.py

```diff
@@ -416,13 +417,29 @@ def module_published_cv(module_org):
     return content_view.read()
 
 
+@pytest.fixture(scope='module')
+def module_promoted_cv(module_org, module_lce, module_published_cv):
+    content_view = module_published_cv
```

Comment:
Why re-name this fixture value?

Reply:
@mshriver, I cannot remember, I guess while testing and experimenting with fixtures
Changes made, please review.